requirements.txt

```diff
@@ -1,22 +1,48 @@
+attrs==19.3.0
```

Comment:
Do you really need that much libraries ?

Reply:
I second this. Some added libs are used for other servies, but what about all the google stuff?